test/Microsoft.Dnx.TestHost.Tests/TestHostTest.cs

```diff
@@ -21,11 +21,11 @@ public class TestHostTest
 
         public TestHostTest()
         {
-            var libraryManager = PlatformServices.Default.LibraryManager;
-            _testProject = Path.GetDirectoryName(libraryManager.GetLibrary("Sample.Tests").Path);
+            var testProjectFolder = Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(Directory.GetCurrentDirectory())));
+            _testProject = Path.Combine(testProjectFolder, "Sample.Tests");
         }
 
-        [ConditionalFact]
+        [ConditionalFact(Skip = "Doesn't support dotnet")]
         [OSSkipCondition(OperatingSystems.Linux)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
Make tests changed in this class skipped `[Fact]`s. The conditions only confuse.